src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/configuration/configuration-form/configuration-form.component.html

```diff
@@ -0,0 +1,148 @@
+<div class="col-sm-12 col-lg-6">
+  <form name="configForm"
+        class="form-horizontal"
+        #formDir="ngForm"
+        [formGroup]="configForm"
+        novalidate>
+    <div class="panel panel-default">
+      <div class="panel-heading">
+        <h3 class="panel-title">
+          <span><span i18n>Edit</span> {{ configForm.getValue('name') }}</span>
+        </h3>
+      </div>
+      <div class="panel-body">
+
+        <!-- Name -->
+        <div class="form-group">
+          <label i18n
+                 class="control-label col-sm-3">Name
+          </label>
+          <div class="col-sm-9">
+            <p class="form-control-static">{{ configForm.getValue('name') }}</p>
+          </div>
+        </div>
+
+        <!-- Description -->
+        <div class="form-group"
+             *ngIf="configForm.getValue('desc')">
+          <label i18n
+                 class="control-label col-sm-3">Description
+          </label>
+          <div class="col-sm-9">
+            <p class="form-control-static">{{ configForm.getValue('desc') }}</p>
+          </div>
+        </div>
+
+        <!-- Long description -->
+        <div class="form-group"
+             *ngIf="configForm.getValue('long_desc')">
+          <label i18n
+                 class="control-label col-sm-3">Long description
+          </label>
+          <div class="col-sm-9">
+            <p class="form-control-static">{{ configForm.getValue('long_desc') }}</p>
+          </div>
+        </div>
+
+        <!-- Default -->
+        <div class="form-group"
+             *ngIf="configForm.getValue('default') !== ''">
+          <label i18n
+                 class="control-label col-sm-3">Default
+          </label>
+          <div class="col-sm-9">
```

Comment:
Please use a read-only input field here.

Reply:
Could you explain why? - In my opinion the "field" is looking a bit nicer that way and there is no situation where you could edit the value atm.

Reply:
> In my opinion the "field" is looking a bit nicer that way

:-) In my opinion it looks extremely ugly.

All forms that come to my mind use read-only input fields to display non editable text.

Reply:
Even if the field itself is never ever going to be used in any case? For me it was just like a "table-like layout helper" so that it fits to the input fields below.

Reply:
> Even if the field itself is never ever going to be used in any case?

Yes

Reply:
Okay, so I will adapt it.

Funny side note: I used read-only fields at first but replaced them by the current solution because it seemed nicer to me :)

Reply:
Updated